Wenab/Wenab.Api/Budgets/GetBudgetController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Options;
+using Wenab.Ynab;
+using Controller = LittleByte.Common.AspNet.Core.Controller;
+
+namespace Wenab.Api.Budgets;
+
+public sealed class GetBudgetController : Controller
+{
+    private readonly YnabOptions options;
+
+    public GetBudgetController(IOptions<YnabOptions> options)
+    {
+        this.options = options.Value;
+    }
+
+    [HttpGet]
+    public async Task<IActionResult> Get()
+    {
+        var client = new HttpClient
+        {
+            DefaultRequestHeaders = { { "Authorization", $"Bearer {options.ApiKey}" } }
```

Comment:
why does this need two brackets

Reply:
Because it's use a `Dictionary` initializer. First set of brackets are for the whole dictionary while the inner brackets are for a single key/value entry.